modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v1/structure.rb

```diff
@@ -90,7 +90,14 @@ def form_title
           '200996'
         end
 
-        def stamp(stamped_pdf_path)
+        def stamp(unstamped_path)
+          stamped_pdf_path = CentralMail::DatestampPdf.new(unstamped_path).run(
```

Comment:
This stamp was previously handled in generator#stamp, which has been removed. Because HLR v1 stamping is handled differently than all other appeals, this portion of the stamping needed to be moved to this method, since it now handles all the stamping for HLR v1.